src/utils/formValidations.js

```diff
@@ -0,0 +1,28 @@
+const validateEmail = (email) => {
+    if (email.length < 1) {
+      return [false, "Email can't be be empty. Enter your email."];
+    }
+    if (!/^.+@.+\..+$/.test(email)) {
```

Comment:
Use a stricter regex because the email will not be strongly validated and the information will be stored in the database.

Reply:
This was fixed with the latest commit: https://github.com/Les-Jailes/Boutique_Clothing/pull/4/commits/e2b34140a6ca0f3bb80d7e7ac5cb0bb0ed37ffc3